site/docs/guides/connect-network.md

```diff
@@ -1,8 +1,9 @@
 # Configure connections with SSH tunneling
 
-Depending on your enterprise network security, you may need to use [SHH tunneling](https://www.ssh.com/academy/ssh/tunneling/example#local-forwarding), or port forwarding, to allow Flow
-to securely access your endpoint. This is configured within a capture or materialization definition, but
-before you can do this, you'll need a properly configured SSH server on your internal network or cloud hosting platform.
+Flow connects to certain types of endpoints — generally databases — using their IP address and port.
+To keep this connection secure, you can configure [SHH tunneling](https://www.ssh.com/academy/ssh/tunneling/example#local-forwarding), or port forwarding.
```

Comment:
Perhaps "For added security" rather than "To keep this connection secure". In many cases the connection is plenty secure even in the absence of SSH tunneling, and it's just an added layer of defense-in-depth.